configs/config.toml.SAMPLE

```diff
@@ -42,7 +42,7 @@
 	# balance below which the client will start reporting errors in logs.
 	# The value should be passed in [wei].
 	#
-	# BalanceAlertThreshold = 500000000000000000 # 0.5 ETH (default value)
+	# BalanceAlertThreshold = "0.5 ether" # 0.5 ether (default value)
```

Comment:
Let's tweak docs a little. `config.toml.SAMPLE` is the only documentation for the client configuration that exists now and we need to be very explicit here. 

We should explain this value can be passed in `wei`, `Gwei`, or `ether` with some examples, just like in the PR description for https://github.com/keep-network/keep-common/pull/60. We should also remove `# The value should be passed in [wei].`  comment as it is no longer up to date.